Automattic/wp-calypso
# Activity Log: Add components and dummy data

This adds the two main components used for this UI, ported from #10862

`ActivityLogItem` - The component for the individual event logged
`ActivityLogDate` - A foldable card component that is the home of all of the daily events within it. 

This PR is using a hard-coded array of events, for display purposes. It is not connected to the state yet nor does it fire any actions. 

To Test: 
- On any Jetpack site, view the `/stats/activity` tab.  
- Observe the dummy items 

<img width="916" alt="screen shot 2017-05-30 at 12 40 17 pm" src="https://cloud.githubusercontent.com/assets/7129409/26594309/2b57ed36-4535-11e7-865c-f0e9e8e0fed2.png">
